### DISCUSSION TITLE
Local deployment errors

### DISCUSSION THREAD

**Discussion by markzhou-github** [mckaywrigley/chatbot-ui#1665] [Help]
I am deploying this on my laptop. After npm run chat, I got below feedback from screen. Please help me how to understand and solve below issues. 

node:internal/modules/cjs/loader:1143
  const err = new Error(message);
              ^

Error: Cannot find module './lib/source-map-generator'
Require stack:

**Comment by tallessouza** [comment]
Did you installed npm modules with npm install?

**Reply by markzhou-github** [reply]
I found the npm modules are not completed installed. problem solved. Thanks

**Comment by Jonneal3** [comment]
@markzhou-github @tallessouza join the discord ---https://discord.gg/K5hkY333kz
